data/aaditya-paul.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Aaditya Paul",
+  "type": "personal",
+  "bio": "I'm a 17 year old full stack web and app developer, ethical hacker and a student hailing from India, West Bengal. And... 𝕴 𝖑𝖔𝖛𝖊 𝖙𝖔 𝖈𝖔𝖉𝖊 ❤️",
+  "avatar":"https://github.com/aaditya-paul.png",
```

Comment:
Hi @aaditya-paul, thank you for contributing  :tada: 
You can delete this line, it is not required anymore  :+1: 
```suggestion
```